.circleci/scripts/npm-audit-check.js

```diff
@@ -13,7 +13,7 @@ for (const advisory of advisories) {
     continue
   }
 
-  count += advisory.findings.some((finding) => (!finding.dev && !finding.optional))
+  count += 1
```

Comment:
Hmm... this isn't quite correct. This is counting advisories, not vulnerable packages.